picard/ui/infodialog.py

```diff
@@ -38,8 +38,7 @@
     encode_filename,
     format_time,
     htmlescape,
-    union_sorted_lists,
-    webbrowser2,
+    union_sorted_lists
```

Comment:
The comma at the end is intentional, please keep it.

Reply:
@phw : damned, you merged even before i review... ;)

Reply:
Sorry, missed the comma :(

Reply:
Oops, I didn't notice I removed it

Reply:
The PR is already merged, should I create a new PR just for that? or will it be faster if any of you adds it directly?

Reply:
I'll fix it

Reply:
Thanks for the patch, @antlarr :)